packages/calcite-components/conventions/Styling.md

```diff
@@ -41,22 +43,31 @@ In the [global CSS file](https://github.com/Esri/calcite-design-system/blob/main
 </div>
 ```
 
-This will cause both the button and the date picker to use the dark mode color variables declared in the global file. This makes it very easy for developers to move an entire app from light to dark mode and vice versa.
-
 To make this work, inside a component's SASS file, _you must use colors from the theme variables_. For example
 
 ```scss
-// 🙅‍♀️ using the sass var will not correctly inherit or change in light/dark mode
+@import "~@esri/calcite-design-tokens/dist/scss/light";
+
+// Using the sass var will set the color of host to the value of the brand color in light mode.
+// 🙅‍♀️ However, it will not correctly inherit or change it's value when swapping light/dark mode.
```

Comment:
I wouldn't call this a non-recommended approach. the `light.scss` file is a new output asset which may be useful in some edge cases. I just wanted to make it clear how it will work.

Reply:
Ah, gotcha. Maybe this example could be placed underneath the other example for clarity - WDYT?